packages/migrate/src/__tests__/Baseline.test.ts

```diff
@@ -0,0 +1,115 @@
+import { jestConsoleContext, jestContext } from '@prisma/internals'
+import fs from 'fs-jetpack'
+import prompt from 'prompts'
+
+import { DbPull } from '../commands/DbPull'
+import { MigrateDeploy } from '../commands/MigrateDeploy'
+import { MigrateDev } from '../commands/MigrateDev'
+import { MigrateResolve } from '../commands/MigrateResolve'
+
+const ctx = jestContext.new().add(jestConsoleContext()).assemble()
+
+// Disable prompts
+process.env.GITHUB_ACTIONS = '1'
+// Disable generate
+process.env.PRISMA_MIGRATE_SKIP_GENERATE = '1'
```

Comment:
Is this isolated to a single test? global variables are always scary

Reply:
Should be only for this test file yes